control_data_collecting_tool/scripts/data_collecting_trajectory_publisher.py

```diff
@@ -410,22 +749,52 @@ def updateNominalTargetTrajectory(self):
 
         long_side_margin = 5
         long_side_margin = 5
-        total_distance = ld * (1 + np.pi) * 2
+        # total_distance = ld * (1 + np.pi) * 2
```

Comment:
please remove commentout if it is not necessary.

Reply:
Thank you for your feedback. I removed commentout.